packages-native/safe-area-view/src/SafeAreaView.tsx

```diff
@@ -10,10 +10,8 @@ export const SafeAreaView = (props: SafeAreaViewProps<SafeAreaViewStyle>): JSX.E
     const styles = flattenStyles(defaultSafeAreaViewStyle, customStyles);
 
     return (
-        <Fragment>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaTop.backgroundColor }} />
-            <ReactSaveAreaView style={{ ...styles.container, flex: 1 }}>{props.content}</ReactSaveAreaView>
-            <ReactSaveAreaView style={{ flex: 0, backgroundColor: styles.unsafeAreaBottom.backgroundColor }} />
-        </Fragment>
+        <ReactSaveAreaView style={{ flex: 1 }}>
+            <View style={{ ...styles.container }}>{props.content}</View>
```

Comment:
dont spread the object.. you dont need it

Reply:
Oops, you're right. Changed it.